packages/app/src/cli/services/app-logs/poll-app-logs.ts

```diff
@@ -145,8 +143,22 @@ export const pollAppLogs = async ({
     }, POLLING_INTERVAL_MS)
     // eslint-disable-next-line no-catch-all/no-catch-all
   } catch (error) {
-    stdout.write(`Error while retrieving app logs.`)
-    stdout.write('App log streaming is no longer available in this `dev` session.')
+    outputWarn(`Error while polling app logs.`)
+    outputWarn(`Retrying in ${POLLING_ERROR_RETRY_INTERVAL_MS / 1000} seconds.`)
     outputDebug(`${error}}\n`)
+
+    setTimeout(() => {
```

Comment:
I think we should extract a reusable method to invoke `pollAppLogs` since we use it here and on line `60`. Then we can also test is standalone.

Reply:
Chatted on slack about this. 
- the setTimeouts are already just: `setTimeout { pollAppLogs {...}, delay}` . It’s essentially a 1 liner already.
- If I create a new retryPolling function, I just end up having to pass all the args around.